src/main/kotlin/com/openlattice/users/Auth0SyncService.kt

```diff
@@ -64,6 +53,34 @@ class Auth0SyncService(
         syncUserEnrollmentsAndAuthentication(user)
     }
 
+    fun createOrUpdateUsers(usersToUpdate: Collection<User>) {
```

Comment:
Something I realized when looking at this fresh is that users need to be added as members of an organization in the organization and the principal service. Need to ensure this is fail safe.